app/views/admin/users/_member_index.html.erb

```diff
@@ -54,9 +59,32 @@
         </tr>
       </thead>
       <tbody class="crayons-card">
-        <% @users.each do |user| %>
+        <% @users.includes([:organizations]).each do |user| %>
         <tr>
-          <td><%= link_to user.name, admin_user_path(user) %></td>
+          <td>
+          <div class="flex s:items-start flex-col s:flex-row">
+            <a href="<%= admin_user_path(user.id) %>">
+              <img class="radius-full align-middle" src="<%= user.profile_image_url_for(length: 50) %>" width="40" height="40" alt="<%= user.name %>" />
+            </a>
+            <div class="pl-3 flex-1 fs-s"><%= user.name %>
+              <p class="color-base-60">@<%= user.username %>&nbsp;
+                <button class="c-pill ml-2 crayons-tooltip__activator" aria-disabled="true"><%= user.current_roles %>
+                  <span data-testid="tooltip" class="crayons-tooltip__content"><%= user.current_roles %></span>
+                </button>
```

Comment:
I took at peek at the designs for this with fresh eyes and noticed that these role tags are supposed to also be tooltips. This being said, I had to rework this code and now I'm struggling to get it to look like it does in the design. 😅

Reply:
I don't think we want to use `button`/`c-pill` in this context. Although there's some similarities in the styling, I see this is as more closely aligned to the `indicator` component, and I don't think we want to introduce buttons which don't have associated actions 😞

Reply:
Agreed! I was trying to implement it closest to the style in the figma for this work, while also following the Crayons documentation and wasn't sure which was the best approach. 😕  This being said, I've reverted the button change and have it back as a `<span>` with the `c-indicator` class, _but_ the tooltip isn't working as expected now--it shows the member's role as expected, but it appears towards the bottom of the screen. 🙃